src/pudl/analysis/service_territory.py

```diff
@@ -255,22 +256,41 @@ def get_territory_geometries(
     )
 
 
-def _save_geoparquet(gdf, entity_type, dissolve, limit_by_state):
-    # For filenames based on input args:
-    dissolved = ""
-    if dissolve:
-        dissolved = "_dissolved"
-    else:
-        # States & counties only remain at this point if we didn't dissolve
-        for col in ("county_id_fips", "state_id_fips"):
-            # pandas.NA values are not compatible with Parquet Strings yet.
-            gdf[col] = gdf[col].fillna("")
-    limited = ""
-    if limit_by_state:
-        limited = "_limited"
-    # Save the geometries to a GeoParquet file
-    fn = f"{entity_type}_geom{limited+dissolved}.pq"
-    gdf.to_parquet(fn, index=False)
```

Comment:
This got much simpler since we last touched it, and now it kind of "just works" with a small number of records in each row group.  Not sure what the memory blowup was, but it's gone now.